samza-core/src/main/java/org/apache/samza/operators/impl/OperatorImpl.java

```diff
@@ -61,6 +62,10 @@ public final void init(Config config, TaskContext context) {
       throw new IllegalStateException(String.format("Attempted to initialize Operator %s more than once.", opName));
     }
 
+    if (closed) {
+      throw new IllegalStateException(String.format("Attempted to initialize Operator %s after it was closed.", opName));
```

Comment:
Mmm.. What do you want it to be updated to? Is n't it already descriptive enough?